### PR TITLE
Contrib/aburt demisto aburt trendmicrodda

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
fixes proxy param typo

### CODE REVIEW

**Comment by ronykoz** [comment]
@bakatzir the hooks are failing due to old checks of the Beta integrations, will fix them on Sunday

**Comment by ronykoz** [comment]
@yaakovi we have some weird behaviour of the hooks here, please take a look

File: Beta_Integrations/integration-TrendMicroDDA_CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## [Unreleased]
-
+Fixed a typo on the proxy parameter.
```
Comment (amshamah419) [inline Beta_Integrations/integration-TrendMicroDDA_CHANGELOG.md]:
```suggestion
Fixed a typo in the proxy parameter.
```

**Comment by ronykoz** [comment]
Force merging due to hooks problems, not relevant to the beta integration

**Pull request merged**